frontend/pubspec.yaml

```diff
@@ -29,7 +29,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
+  firebase_core: "0.7.0"
```

Comment:
To fix the null safety issue, I updated these to:

firebase_core: "^1.10.0",
cloud_firestore: "^3.1.0"